man/adoc/bpftrace.adoc

```diff
@@ -1993,6 +1993,12 @@ For string arguments in an action block use the `str()` call to retrieve the val
 | get_current_uid_gid
 | User name of the current thread, as seen from the init namespace
 
+| `usermode`
```

Comment:
maybe this should be `is_user_mode` or `in_user_mode` ? WDYT?

Reply:
I prefer `usermode` myself. Nice and short and fits better with the existing builtins, which don't use underscores

Reply:
`usermode` does fit with the current builtins better but I like an `is` or `in` to indicate its giving a boolean-ish value. Don't feel strongly though.